src/components/Guilds/Editor/index.tsx

```diff
@@ -0,0 +1,168 @@
+import styled, { css } from 'styled-components';
+import { transparentize } from 'polished';
+import { useEditor, EditorContent } from '@tiptap/react';
+import StarterKit from '@tiptap/starter-kit';
+import Focus from '@tiptap/extension-focus';
+import Highlight from '@tiptap/extension-highlight';
+import MenuBar from './MenuBar';
+import TurndownService from 'turndown';
+
+var turndownService = new TurndownService();

```

Comment:
Do we have to use `var`?

Reply:
Copy and pasted from some docs :P good spot